src/reducers/clear.js

```diff
@@ -14,10 +16,16 @@ import {
 // SIDE EFFECTS: localStorage, scroll
 // preserves some settings
 export const clear = state => {
-  localStorage.clear()
-  localStorage['settings-dark'] = state.settings.dark
-  localStorage['settings-tutorialStep'] = TUTORIAL_STEP_NONE
-  localStorage['helper-complete-welcome'] = true
+  localForage.clear().then(() => {
+    const promises = [
+                      localForage.setItem('settings-dark', state.settings.dark),
+                      localForage.setItem('settings-tutorialStep', TUTORIAL_STEP_NONE),
+                      localForage.setItem('helper-complete-welcome', true)
+                     ]
+    return Promise.all(promises)
```

Comment:
Please avoid single-use variables. Instead, inline the array of promises in `Promise.all(...)`.